### PR TITLE
[4.0] CLI help text

### PR DESCRIPTION
**Pull request by brianteeman**
Try to make the help text consistent in code, grammar and usage

` php cli\joomla.php user:removefromgroup`

### CODE REVIEW

File: libraries/src/Console/SetConfigurationCommand.php [hunk 1]
```diff
@@ -243,19 +243,17 @@ public function saveConfiguration($options): bool
...
 		);
 
-		$help = "The <info>%command.name%</info>
-				Sets a value for a configuration option
+		$help = "<info>%command.name%</info> sets the value for a configuration option
 				\nUsage: <info>php %command.full_name%</info> <option>=<value>";
 
-		$this->setHelp($help);
+			$this->setDescription('Set a value for a configuration option');
+			$this->setHelp($help);
```
Comment (Quy) [inline libraries/src/Console/SetConfigurationCommand.php]:
```suggestion
		$this->setDescription('Set a value for a configuration option');
		$this->setHelp($help);
```

File: libraries/src/Console/CheckJoomlaUpdatesCommand.php [hunk 2]
```diff
@@ -47,11 +47,9 @@ class CheckJoomlaUpdatesCommand extends AbstractCommand
 	 */
 	protected function configure(): void
 	{
-		$help = <<<'EOF'
-The <info>%command.name%</info> Checks for Joomla updates.
+		$help = "<info>%command.name%</info> will check for Joomla updates
+\nUsage: <info>php %command.full_name%</info>";
```
Comment (Quy) [inline libraries/src/Console/CheckJoomlaUpdatesCommand.php]:
```suggestion
		\nUsage: <info>php %command.full_name%</info>";
```

File: libraries/src/Console/GetConfigurationCommand.php [hunk 3]
```diff
@@ -308,17 +308,16 @@ protected function configure(): void
...
 
-		$help = "The <info>%command.name%</info> Displays the current value of a configuration option
+		$help = "<info>%command.name%</info> displays the current value of a configuration option
 				\nUsage: <info>php %command.full_name%</info> <option>
 				\nGroup usage: <info>php %command.full_name%</info> --group <groupname>
 				\nAvailable group names: $groupNames";
 
-		$this->setHelp($help);
+			$this->setDescription('Display the current value of a configuration option');
+			$this->setHelp($help);
```
Comment (Quy) [inline libraries/src/Console/GetConfigurationCommand.php]:
```suggestion
		$this->setDescription('Display the current value of a configuration option');
		$this->setHelp($help);
```

File: libraries/src/Console/ListUserCommand.php [hunk 4]
```diff
@@ -113,13 +113,11 @@ private function configureIO(InputInterface $input, OutputInterface $output)
...
 	protected function configure(): void
 	{
+		$help = "<info>%command.name%</info> will list all users
+		\nUsage: <info>php %command.full_name%</info>";
+
 		$this->setDescription('List all users');
-		$this->setHelp(
-			<<<EOF
-The <info>%command.name%</info> command lists all users
+		$this->setHelp($help);
```
Comment (Quy) [inline libraries/src/Console/ListUserCommand.php]:
```suggestion
```

File: libraries/src/Console/ExtensionInstallCommand.php [hunk 5]
```diff
@@ -88,21 +88,16 @@ protected function configure(): void
 		$this->addOption('path', null, InputOption::VALUE_REQUIRED, 'The path to the extension');
 		$this->addOption('url', null, InputOption::VALUE_REQUIRED, 'The url to the extension');
 
-		$this->setDescription('Install an extension from a URL or from a path');
-
-		$help = <<<'EOF'
-The <info>%command.name%</info> is used to install extensions
-
-  <info>php %command.full_name%</info>
```
Comment (richard67) [inline libraries/src/Console/ExtensionInstallCommand.php]:
```suggestion
```
Delete line 91 for not having 2 empty lines in a row, which currently makes Drone PHPCS fail.

**Comment by bonzani** [comment]
I have tested this item :white_check_mark: successfully on ab85a48806c045fd50e2e531f1134c3bf749cb3c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29811">issues.joomla.org/tracker/joomla-cms/29811</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on ab85a48806c045fd50e2e531f1134c3bf749cb3c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29811">issues.joomla.org/tracker/joomla-cms/29811</a>.</sub>

**Comment by alikon** [comment]
rtc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29811">issues.joomla.org/tracker/joomla-cms/29811</a>.</sub>

**Comment by richard67** [comment]
Thanks!

**Pull request merged**